force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,24 +418,50 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
-        
+        private String name;
+        private Integer age;
         // Constructor to initialize the name and age
-    
+        public Person(String name,Integer year)
```

Comment:
I recommend naming your age variable `age` and not `year.`

Reply:
That is clearer. Thanks.